src/pages/module/ssl/edit/index.tsx

```diff
@@ -11,21 +11,37 @@ import {
   update as updateSSL,
 } from '@/services/ssl';
 import { useForm } from 'antd/es/form/util';
+import { router } from 'umi';
 
 const layout = {
-  wrapperCol: { span: 8 },
+  labelCol: {
+    span: 2,
+  },
+  wrapperCol: {
+    span: 8,
+  },
 };
 
-const Detail: React.FC = () => {
+const tailLayout = {
+  wrapperCol: {
+    offset: 2,
+  },
+};
+
+const Edit: React.FC = () => {
   const [mode] = useState<PageMode>(getPageMode());
   const { key } = useParams();
   const [form] = useForm();
+  const hideLoading = message.loading(formatMessage({ id: 'component.global.loading' }), 0);
 
   useEffect(() => {
     if (mode === 'EDIT' && key) {
       fetchSSLItem(key).then(data => {
         form.setFieldsValue(data.value);
+        hideLoading();
```

Comment:
?

Reply:
I added a load message for the original data load of the edit page. (by antd's message component)
It was originally planned to add load message when submitting, but it has not been completed yet.

Reply:
ok